sunflower/plugin_base/item_list.py

```diff
@@ -1406,13 +1406,18 @@ def _edit_selected(self, widget=None, data=None):
 	def _inherit_left_path(self, widget, data=None):
 		"""Inherit path in right list from left"""
 		opposite_object = self._parent.get_opposite_object(self)
+		selection = self._get_selection()
+		if ((selection is not None) and (self.get_provider().is_dir(selection))):
```

Comment:
Brackets are not needed in conditions. This could be written like this:
```python
if selection and self.get_provider().is_dir(selection):
```

Reply:
Thank you. Just pushed a fixed version.